src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -7,6 +7,7 @@ import { currentVenueSelectorData } from "utils/selectors";
 import { useInterval } from "hooks/useInterval";
 import { useSelector } from "hooks/useSelector";
 import { usePartygoers } from "hooks/users";
+import { useExperiences } from "hooks/useExperiences";
```

Comment:
We probably want to rename this to `useExperience` as a followup to this PR at some stage too.